controlplane/src/core/bufservices/PlatformService.ts

```diff
@@ -9468,6 +9468,7 @@ export default function (opts: RouterOptions): Partial<ServiceImpl<typeof Platfo
               details: `Organization not found`,
             },
             organizationName: '',
+            organizationSlug: '',
```

Comment:
Because organization name is not unique, could you use slug also for the organization select list when the user logs in?